front/.eslintrc.js

```diff
@@ -115,7 +115,7 @@ module.exports = {
       },
     ],
     "@typescript-eslint/consistent-type-imports": ["error", { "prefer": "no-type-imports" }],
-    'no-console': ['error', { allow: ['group', 'groupCollapsed', 'groupEnd'] }],
+    'no-console': 'off'
```

Comment:
Do we really want to disable this rule?

Reply:
I'll leave it activated for now but we'll create a ticket to discuss it because it slows down the DX.

Reply:
I put it in 'warn' only, so we still have a message in the linter but we don't need to add an exception.

I created this ticket https://github.com/twentyhq/twenty/issues/2238